traffic_control/tests/test_api.py

```diff
@@ -0,0 +1,89 @@
+import datetime
+
+from django.contrib.gis.geos import Point
+from django.urls import include, path, reverse
+from rest_framework import status
+from rest_framework.test import APITestCase, URLPatternsTestCase
+
+from traffic_control.models import Lifecycle, TrafficSignCode, TrafficSignPlan
+from users.models import User
+
+
+class TrafficSignPlanTests(APITestCase, URLPatternsTestCase):
+    urlpatterns = [
+        path("", include("city-infrastructure-platform.urls")),
+    ]
+
+    def test_get_all_traffic_sign_plans(self):
+        """
+        Ensure we can get all traffic sign plan objects.
+        """
+        self.user = User.objects.create_user(username="testuser", password="12345")
+        lifecycle = Lifecycle.objects.create(status="ACTIVE", description="Active")
+        code = TrafficSignCode.objects.create(code="A11", description="Tietyo")
+        TrafficSignPlan.objects.create(
+            code=code,
+            location_xy=Point(25496366.48055263, 6675573.680776692, srid=3879),
```

Comment:
maybe using settings variable instead of hard-coded srid?

Reply:
Updated